terraform/central-account/lb.tf

```diff
@@ -27,6 +27,39 @@ resource "aws_lb_target_group_attachment" "test" {
   port             = 8081
 }
 
+resource "aws_lb_listener_rule" "unauthenticated-routes-1" {
```

Comment:
I suggest adding a one-line comment for why we'd want to allow direct access to these routes. Something like "The CLI needs to access these routes directly on the server without going through ALB's auth"

Reply:
Added